deepspeed/runtime/zero/stage_1_and_2.py

```diff
@@ -535,6 +538,12 @@ def _create_param_mapping(self):
 
         return param_mapping
 
+    def _uncreate_param_mapping(self):
```

Comment:
This should be a utility function in [mixed_precision_linkage.py](https://github.com/microsoft/DeepSpeed/blob/master/deepspeed/utils/mixed_precision_linkage.py) as inverse of [link_hp_params](https://github.com/microsoft/DeepSpeed/blob/0e0748c579cff3f94dc0609677623d082c9a2132/deepspeed/utils/mixed_precision_linkage.py#L11).